climada/engine/uncertainty_quantification/input_var.py

```diff
@@ -66,8 +66,7 @@ class InputVar():
     Categorical variable function: LitPop exposures with m,n exponents in [0,5]
         import scipy as sp
         def litpop_cat(m, n):
-            exp = Litpop()
-            exp.set_country('CHE', exponent=[m, n])
+            exp = Litpop.from_countries('CHE', exponent=[m, n])
```

Comment:
This is a minimal example of the anti-pattern that we want to avoid by introducing the static constructors.